package.json

```diff
@@ -6,7 +6,11 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "lint:py": "flake8 --exit-zero",
-    "makemessages": "node ./extract_frontend_messages.js ./contentcuration/contentcuration/static/js/edit_channel/**{/*,}.{js,vue}"
+    "makemessages": "node ./extract_frontend_messages.js ./contentcuration/contentcuration/static/js/edit_channel/**{/*,}.{js,vue}",
+    "build": "webpack --config webpack.config.js",
+    "minio": "minio server ~/.minio_data/",
```

Comment:
@aronasorman included minio in `yarn run devserver` because running it separately became bothersome.